pkg/providers/tinkerbell/tinkerbell.go

```diff
@@ -369,6 +383,15 @@ func buildTemplateMapCP(clusterSpec *cluster.Spec, controlPlaneMachineSpec v1alp
 		"kubeVipImage":                 "ghcr.io/kube-vip/kube-vip:latest",
 		"podCidrs":                     clusterSpec.Spec.ClusterNetwork.Pods.CidrBlocks,
 		"serviceCidrs":                 clusterSpec.Spec.ClusterNetwork.Services.CidrBlocks,
+		"BaseRegistry":                 "", // TODO: need to get this values for creating template IMAGE_URL
+		"OSDistro":                     "", // TODO: need to get this values for creating template IMAGE_URL
+		"OSVersion":                    "", // TODO: need to get this values for creating template IMAGE_URL
+		"KubernetesVersion":            "", // TODO: need to get this values for creating template IMAGE_URL
```

Comment:
camelCase for consistency
```suggestion
		"baseRegistry":                 "", // TODO: need to get this values for creating template IMAGE_URL
		"osDistro":                     "", // TODO: need to get this values for creating template IMAGE_URL
		"osVersion":                    "", // TODO: need to get this values for creating template IMAGE_URL
		"kubernetesVersion":            "", // TODO: need to get this values for creating template IMAGE_URL
```